src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CreateCaseCallbackService.java

```diff
@@ -100,30 +110,56 @@ public CreateCaseCallbackService(
         ExceptionRecord exceptionRecord,
         ServiceConfigItem configItem,
         long caseId,
-        boolean ignoreWarnings
+        boolean ignoreWarnings,
+        String idamToken,
+        String userId
     ) {
+        String caseIdAsString = Long.toString(caseId);
+
+        log.info("Start creating new case for {} from exception record {}", configItem.getService(), caseIdAsString);
+
         try {
-            log.info(
-                "Start creating exception record for service {} and exception record {}",
-                configItem.getService(),
-                caseId
-            );
+            String s2sToken = s2sTokenGenerator.generate();
 
             SuccessfulTransformationResponse transformationResponse = transformationClient.transformExceptionRecord(
                 configItem.getTransformationUrl(),
                 exceptionRecord,
-                s2sTokenGenerator.generate()
+                s2sToken
             );
 
             if (!ignoreWarnings && !transformationResponse.warnings.isEmpty()) {
-                log.warn("Transformation warnings: {}", String.join(", ", transformationResponse.warnings));
-
-                return Validation.invalid(Array.ofAll(transformationResponse.warnings));
-            } else {
+                // do not log warnings
                 return Validation.valid(new ProcessResult(
```

Comment:
I think we don't have any `Validation.invalid` case anymore, so this method can return ProcessResult. Can be done in a future PR.

Reply:
Yep. part of whichever cleanup I'm planning to do. Ta